src/web-server.js

```diff
@@ -285,9 +285,12 @@ app.delete("/services/:serviceName", function (req, res) {
 
 let server;
 if (process.env.NODE_ENV !== "test") {
-  server = app.listen(getConfig().port || 5000, function () {
-    console.log("Listening at http://localhost:%s", server.address().port);
-  });
+  server = app.listen(
+    process.env.PORT || getConfig().port || 5000,
```

Comment:
I think it's a common pattern to prioritize environment variables over configuration files??

Reply:
that is a good question haha. I honestly can't remember

Reply:
seems logical though now that I think about it